docs/ml-features.md

```diff
@@ -1389,3 +1389,145 @@ print(output.select("features", "clicked").first())
 
 # Feature Selectors
 
+## VectorSlicer
+
+`VectorSlicer` is a transformer that takes a feature vector and outputs a new feature vector with a sub-array of the original features. It is useful for extracting features from a vector column.
+
+`VectorSlicer` accepts a vector column with a specified indices, then outputs a new vector column whose values are selected via those indices. There are two types of indices, 
+
+ 1. Integer indices that represents the real indices in the vector, `setIndices()`;
```

Comment:
I would remove the word "real" (i.e. "...that represent the indices into the vector") since it could be confused for real numbers (i.e. real-valued indices, which don't really make sense)